Source/Meadow.Foundation.Core/Leds/RgbLedColors.cs

```diff
@@ -1,41 +1,44 @@
-﻿namespace Meadow.Foundation.Leds
+﻿namespace Meadow.Foundation.Leds;
+
+/// <summary>
+/// Colors for RGB Led
+/// </summary>
+public enum RgbLedColors
 {
     /// <summary>
-    /// Colors for RGB Led
-    /// </summary>
-    public enum RgbLedColors
-    {
-        /// <summary>
-        /// Red (red LED only)
-        /// </summary>
-        Red,
-        /// <summary>
-        /// Green (green LED only)
-        /// </summary>
-        Green,
-        /// <summary>
-        /// Blue (blue LED only)
-        /// </summary>
-        Blue,
-        /// <summary>
-        /// Yellow (red and green LEDs)
-        /// </summary>
-        Yellow,
-        /// <summary>
-        /// Magenta (blue and red LEDs)
-        /// </summary>
-        Magenta,
-        /// <summary>
-        /// Cyan (blue and green LEDs)
-        /// </summary>
-        Cyan,
-        /// <summary>
-        /// White (red, green and blue LEDs)
-        /// </summary>
-        White,
-        /// <summary>
-        /// Count of colors
-        /// </summary>
-        count,
-    }
+    /// Red (red LED only)
+    /// </summary>
+    Red,
+    /// <summary>
+    /// Green (green LED only)
+    /// </summary>
+    Green,
+    /// <summary>
+    /// Blue (blue LED only)
+    /// </summary>
+    Blue,
+    /// <summary>
+    /// Yellow (red and green LEDs)
+    /// </summary>
+    Yellow,
+    /// <summary>
+    /// Magenta (blue and red LEDs)
+    /// </summary>
+    Magenta,
+    /// <summary>
+    /// Cyan (blue and green LEDs)
+    /// </summary>
+    Cyan,
+    /// <summary>
+    /// White (red, green and blue LEDs)
+    /// </summary>
+    White,
+    /// <summary>
+    /// Black (all LEDs off)
+    /// </summary>
+    Black,
```

Comment:
We removed black because it breaks the paradigm we have for the other LEDs, so the only way to turn off an led is with the IsOn boolean.